### DISCUSSION TITLE
Yarra Server for Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by liamtimms** [yarra-framework/yarra-framework.github.io#3] [Q&A] [answered]
Hi,

We are looking at implementing YarraServer in our workflows, and I am currently using a machine running Ubuntu 22.04 LTS. I see here https://yarra-framework.org/server/installation/ that 

>  currently supported operating systems for YarraServer are Ubuntu 16.04 LTS 64bit and Ubuntu 18.04 LTS 64bit.

I was curious if 

a) there are plans to release a 22.04 LTS 64bit version soon or 

b) if there are any plans to distribute a Docker image of the server using an Ubuntu 18.04 LTS base?

Thanks!

**Comment by tblock79** [comment, marked as answer]
Hi!

The current version of YarraServer runs on Ubuntu 22.04 as well. We just migrated all our servers to 22.04. If you still have Windows XP-based MRI scanners (Avanto, Verio, Espree) it is necessary to explicitly enable support for the SAMBA SMBv1 protocol because Ubuntu 22.04 has support for that protocol disabled by default. Otherwise, it should work out of the box, I think.

Best,
  Tobias

**Reply by liamtimms** [reply]
Great thanks, I'll test it on 22.04.
